ReverseString.cls

```diff
@@ -0,0 +1,10 @@
+public class ReverseString {
+	public static void rString(String testString){
+		
+		String revString=testString.reverse();
```

Comment:
@Eternus-noopur I don't think you should use String function here. I guess you shold write logic for string reverse.